b2b-sample/README.md

```diff
@@ -159,22 +159,12 @@ setup the config json file as mentioned above.
 
 ### Step 6: Setup the identity provider for the Guardio-Business-App
 
-* For this we will create an external identity provider in Asgardeo. After creating an identity provider in Asgardeo.
-
-* Also, On the User Attributes tab, click on + Add User Attributes.
-    * Select `Email`, `First Name`, `Last Name`, and `Username` from the list of attributes.
-
+* For this we will create an external identity provider in sub organization.
 * Go to the identity providers section in the **Guardio-Admin-App** and fill in the form to create an identity provider for the
   **Guardio-Business-App**.
 
 * After creating the Idp add it to the login flow by clicking on `Add to Login Flow`.
 
-### Step 7: Create a corporate user
-* You need to create a new corporate user to access the **Guardio-Business-App**.
-
-* To create a corporate user for Best Auto Mart organization go to the Asgardeo console where you set up the corporate IDP
-  and create a user named `John`.
-
```

Comment:
Business user will be log in via the Enterprise IDP. Even after configuring the enterprise IDP, the basic authentication option will be removed. In such cases, there is no point of creating a business user in the sub organization itself.